src/constants.js

```diff
@@ -217,6 +217,29 @@ const DESCRIPTION_CITY = [
 const MINUTES_IN_DAY = 24*60;
 const HOUR = 60;
 
+const SortType = {
+  DAY: {
+    name: 'day',
+    disabled: false,
+  },
+  EVENT: {
+    name: 'event',
+    disabled: true,
+  },
+  TIME: {
+    name: 'time',
+    disabled: false,
+  },
+  PRICE: {
+    name: 'price',
+    disabled: false,
+  },
+  OFFERS: {
+    name: 'offers',
+    disabled: true,
+  },
+};
```

Comment:
```suggestion
const SortType = {
  DAY: 'day',
};
```
Делаем так. А дальше то что дизейбленное, будем считать там где нам нужно

Reply:
вот тут вопрос, я же sort-view переписал и там в верстку подставляется {name}, если сразу без общего имени, а со значениями объект делать, то как передать в сорт-вье общее имя ключа?

Reply:
Так ты когда сделаешь `Object.values(SortType)`, то у тебя будет сразу массив имен (а не как сейчас, где есть объект с ключом name)